src/renderers/webgl/WebGLTextures.js

```diff
@@ -1763,7 +1798,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 				_gl.bindRenderbuffer( _gl.RENDERBUFFER, null );
 
-				if ( renderTarget.depthBuffer ) {
+				if ( renderTarget.depthBuffer && renderTargetProperties.__webglDepthRenderbuffer === undefined ) {
```

Comment:
I don't understand this bit. 

`setupRenderTarget()` is only called when `__webglFramebuffer` is `undefined` (so no framebuffer creation happened so far). When is `__webglFramebuffer`, `undefined` but not `__webglDepthRenderbuffer`?

Reply:
Just removed this. I added it when checking whether the depth buffer needed to be initialized in other functions but it's not needed here.